src/pymodaq/dashboard.py

```diff
@@ -785,8 +787,8 @@ def set_file_preset(self, filename):
                 self.load_layout_state(path)
 
             self.mainwindow.setWindowTitle(f'PyMoDAQ Dashboard: {self.title}')
-            if self.pid_module is not None:
-                self.pid_module.set_module_manager(detector_modules, actuators_modules)
+            # if self.pid_module is not None:
+            #     self.pid_module.set_module_manager(detector_modules, actuators_modules)
```

Comment:
why commenting these?

Reply:
These comments are now removed.
The pid module loading is done in Dashboard.set_preset_mode.